### DISCUSSION TITLE
[Bug] git (bash) aliases

### DISCUSSION THREAD

**Discussion by michahell** [ScoopInstaller/Scoop#5876] [Help and Questions]
#### Current Behavior
when editing the file `C:\...\tools\apps\git\current\etc\profile.d`
and adding aliases, I do not see those getting picked up when starting a new gitBash shell.

#### Expected Behavior
I expect to be able to use my aliases after having copied them into the file:
`C:\...\tools\apps\git\current\etc\profile.d`

#### Additional context/output
using an alias that was copypasted yields:
`bash: gupv: command not found`

`alias` yields nothing (empty output)

#### Possible Solution
I wish I had a solution :(

### System details

**Windows version:** 10

**OS architecture:** 64bit

**PowerShell version:** N/A

**Additional software:** GIT / BASH

#### Scoop Configuration

```json
//# Your configuration here
```

**Comment by rashil2000** [comment]
You'll probably have to read up on how Git for Windows sources its startup scripts, to determine their sourcing order. My recommendation: put your aliases in ~/.profile and then check.
